.rultor.yml

```diff
@@ -0,0 +1,2 @@
+merge:
+  script: echo "Nothing to do (yet)."
```

Comment:
planning to do that later as described, no time right now. We could try
getting the tests to work in rultor which is probably easy and a good pre
merge validation as the CI services don't revalidate a PR when master
changes.

This has btw. the nice side effect that you can do rultor merges from your
phone through github comments ;)

2015-08-05 15:51 GMT+02:00 Erick Wilder <notifications@github.com>:

> In .rultor.yml
> <https://github.com/mitsuhiko/babel/pull/186#discussion_r36303121>:
>
> > @@ -0,0 +1,2 @@
> > +merge:
> > +  script: echo "Nothing to do (yet)."
>
> It worth adding a configuration that does nothing? Maybe you can add a
> simple (but useful) script here?
> This is not a blocking, just a suggestion.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mitsuhiko/babel/pull/186/files#r36303121>.
>